pkg/kubelet/kubelet.go

```diff
@@ -2880,6 +2880,7 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 	}
 	if resizeStatus != "" {
 		kl.statusManager.SetPodResizeStatus(pod.UID, resizeStatus)
+		metrics.ResizeRequestsTotal.WithLabelValues(string(resizeStatus)).Inc()
```

Comment:
Based on @tallclair 's https://github.com/kubernetes/kubernetes/pull/128111#discussion_r1830356807, we should capture the metric inside `statusmanager.SetPodResizeStatus` func here - https://github.com/kubernetes/kubernetes/blob/6cc357046615410434b2fa99c5a229e5db2eb6d9/pkg/kubelet/status/state/state_mem.go#L88

Reply:
Thanks for the remind